src/inpod/config.rs

```diff
@@ -177,8 +177,7 @@ mod test {
     macro_rules! fixture {
         () => {{
             if !crate::test_helpers::can_run_privilged_test() {
```

Comment:
**nit**: privilged is spelled wrong, maybe we can update that since we're in here. Doesn't seem to be called in a lot of places other than the files you're already touching.